### PR TITLE
Fix export to q3d

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: examples/05-Q3D/Q3D_DC_IR.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 import os
 import pyaedt
-
+from pyedb import Edb
```
Comment (PipKat) [inline examples/05-Q3D/Q3D_DC_IR.py:14]:
```suggestion
From PyEDB import EDB
```

Comment (maxcapodi78) [reply examples/05-Q3D/Q3D_DC_IR.py:14]: @PipKat sorry but this code will not wor. Python requires "from" and the exact name of the package

**Review by svandenb-dev — DISMISSED** [review]
LGTM

File: pyaedt/modules/SolveSetup.py [hunk 2]
```diff
@@ -1973,21 +1974,61 @@ def _get_primitives_points_per_net(self):
...
+                    primitive_dict[net].append(pt)
+                    next_p = int(len(points[0]) / 2)
+                    pt = [points[0][next_p], points[1][next_p]]
+                    pt.append(z)
+                    primitive_dict[net].append(pt)
+
+                else:
+                    n = 0
+                    while n < 1000:
+                        n += 10
```
Comment (SMoraisAnsys) [inline pyaedt/modules/SolveSetup.py:2025]: A week ago we had to switch back to `n += 20`. Do your changes handle the reason for that switch back ?

Comment (Samuelopez-ansys) [reply pyaedt/modules/SolveSetup.py:2025]: @SMoraisAnsys This is an algorithm issue. @maxcapodi78 figured out how to make it more robust

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**